src/test/views/test_read_apriori.py

```diff
@@ -153,69 +153,69 @@ def test_returns_expected_data_keys():
     (
         ("species", "CO2"),
         ("latitude", 35),
-        ("path", os.path.join(DATADIR, "apriori_CO2.mat")),
+        ("path", "s3://odin-apriori/apriori_CO2.mat"),
     ),
 )
-def test_returns_expected_data(key, expect):
+def test_returns_expected_data(key, expect, app_context):
     species = "CO2"
     day_of_year = 14
     latitude = 35
     data = read_apriori.get_apriori(
         species,
         day_of_year,
         latitude,
-        datadir=DATADIR,
     )
     assert data[key] == expect
 
 
-def test_returns_expected_pressure():
+def test_returns_expected_pressure(app_context):
     species = "CO2"
     day_of_year = 14
     latitude = 35
     data = read_apriori.get_apriori(
         species,
         day_of_year,
         latitude,
-        datadir=DATADIR,
     )
     np.testing.assert_allclose(
         data["pressure"][:10].tolist(),
         [
             100000.0,
-            93057.204093,
+            97162.795158,
+            94406.087629,
+            91727.593539,
+            89125.093813,
             86596.432336,
-            80584.218776,
-            74989.420933,
-            69783.058486,
-            64938.163158,
-            60429.639024,
-            56234.132519,
-            52329.911468,
+            84139.514165,
+            81752.303794,
+            79432.823472,
+            77179.151559,
```

Comment:
Why did this happen?

Reply:
This is strange, yes, but we did change the source of the apriori file from the one in the repo to the one in "live production".